core/src/main/java/hudson/PluginManager.java

```diff
@@ -2105,7 +2105,9 @@ public String toString() {
     }
     public static boolean FAST_LOOKUP = !SystemProperties.getBoolean(PluginManager.class.getName()+".noFastLookup");
 
+    @Deprecated
```

Comment:
if deprecating please add a javadoc comment saying what developers should use instead